app/components/filters/date-range-filter/date-input.ts

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+
+interface Signature {
+  Element: HTMLInputElement;
```

Comment:
This one uses `...attributes` in the template. The other one doesn't (but maybe I forgot, since we're not using Glint yet).

> The Element field declares what type of element(s), if any, the component applies its passed ...attributes to. 

https://typed-ember.gitbook.io/glint/environments/ember/component-signatures